### ISSUE TITLE
AVT 2 - ComboBox fails WCAG 2.1 for color contrast

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4376]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React

## Detailed Description
Go to the React ComboBox Component
Start CCA and compare the highlighted option text against the menu and it fails 1.4.11.
<img width="844" alt="Screen Shot 2019-10-17 at 2 57 39 PM" src="https://user-images.githubusercontent.com/21676914/67043089-df416b80-f0ee-11e9-987a-bb5eca7309e7.png">
Also, the Menu contrast against the page background also fails.
<img width="836" alt="Screen Shot 2019-10-17 at 2 57 58 PM" src="https://user-images.githubusercontent.com/21676914/67043129-f84a1c80-f0ee-11e9-8a8a-49cc0ce67e77.png">

**Comment by dakahn** [comment]
@aagonzales We should update this token to use the higher contrast too, right?

**Comment by aagonzales** [comment]
> Start CCA and compare the highlighted option text against the menu and it fails 1.4.11.

I don't think you're sampling the correct colors.
`#f4f4f4` is the color of the menu and `#e5e5e` is the color of the hover.

The text on the hover background is: `#161616` ($text-01)
The background hover color is: `#e5e5e5` ($ui-hover)
**That passes at 14.37:1**

> Also, the Menu contrast against the page background also fails.

The drop-shadow is what makes the contrast difference between the page background and the menu. Because of the shadow those two colors don't actually touch. 


I believe these are both non-issues.

**Comment by aagonzales** [comment]
Going to close this because the text is correct contrast ratio and we have a separate issue to research and address the 3:1 non-text contrast issues from the system level.
